### ISSUE TITLE
Links from main page go to versioned pages

### ISSUE THREAD

**Issue by Grinnz** [OpusVL/perldoc.perl.org#41]
Browsing the documentation brings you to versioned URLs. This is a problem for SEO, as it was with search.cpan.org. Google and other search engines will crawl the website and index version specific pages, and five years from now will still be serving 5.30.0 outdated results.

**Comment by JohnMica** [comment]
using 
```
<link rel="canonical" href="/index-functions.html">
```
in the header link should mitigate this issue

**Comment by Grinnz** [comment]
That is a good idea as well but not sufficient. The unversioned URLs should be foremost what people use and copy around.

**Comment by JohnMica** [comment]
there is a 'default' unversioned that will default to the latest stable release (currently that could be 5.30.0)

any other suggestions are appreciated

**Comment by Grinnz** [comment]
Exactly, these unversioned URLs are what should be linked in the menu and on the front page when not starting from a versioned URL -- just like the old site and perldoc.pl.

**Comment by JohnMica** [comment]
i guess because there is no 'unversioned' PERL the default 'unversioned URL search result using Google' ( as an example) is be to link to the latest version released.

- the .default server forwarding will take this into consideration
- if new version released, the .default would point to that new version

**Comment by Grinnz** [comment]
This is now compounded by the main page search going to versioned pages. This is a serious problem.
